cluster/src/main/java/io/scalecube/cluster/gossip/GossipProtocolImpl.java

```diff
@@ -169,8 +168,8 @@ public void stop() {
   }
 
   @Override
-  public Observable<Message> listen() {
-    return subject.onBackpressureBuffer().asObservable();
+  public Flux<Message> listen() {
+    return Flux.from(subject);
```

Comment:
redundant, you may just keep FluxProcessor instead of Processor and return it as is